### ISSUE TITLE
Access-Control-Expose-Headers: X-RateLimit-Limit, X-RateLimit-Remaining

### ISSUE THREAD

**Issue by mattiapontonio** [nasa/apod-api#64]
Can You please ad this http header?

```
Access-Control-Expose-Headers: X-RateLimit-Limit, X-RateLimit-Remaining
```

I wanted to display the data in the footer of my app.
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Expose-Headers

---

Note:
Since I'm using different endpoints from https://api.nasa.gov/, if this is a solution, a good feature, I'm looking to spread the change request across multiple endpoints.

Regards
Mattia

**Comment by kmt29** [comment]
Yes, I'm also running into the same issue. It'll be cool if they could let server expose these two headers. It is also critical that we can monitor our api limits autonomously in our application. Since Expose Header is not set and thus almost every trusted modern browsers are preventing us from reading the api rate limit remaining
